CONTRIBUTING.md

```diff
@@ -8,6 +8,7 @@ Everything you need to know to contribute efficiently to the project.
 
 - [pyrovision](https://github.com/pyronear/pyro-vision/blob/master/pyrovision) - The actual pyrovision library
 - [references](https://github.com/pyronear/pyro-vision/blob/master/references) - Scripts to reproduce performances
+- [docs](https://github.com/frgfm/Holocron/blob/master/docs) - Sphinx documentation building
```

Comment:
Why the reference is on holocron and not [here](https://github.com/pyronear/pyro-vision/tree/master/docs)  ?

Reply:
My bad, wrong copy paste !